lib/webauthn/authenticator_attestation_response.rb

```diff
@@ -35,7 +35,8 @@ def credential
 
     def attestation_statement
       @attestation_statement ||=
-        WebAuthn::AttestationStatement.from(attestation["fmt"], attestation["attStmt"])
+        WebAuthn::AttestationStatement.from(attestation["fmt"], attestation["attStmt"], authenticator_data,
+                                            client_data.hash)
```

Comment:
Sorry for the back and forth, and if my previous comment lead you this way.

Reply:
Makes sense, I changed it. In the end there's no need for the certificate -> attested credential data fallback, since we already check they must be the same if the certificate version is present.